nodejs/packages/layer/src/wrapper.ts

```diff
@@ -1,12 +1,21 @@
 import { NodeTracerConfig, NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
 import {
   BatchSpanProcessor,
+  ConsoleSpanExporter,
   SDKRegistrationConfig,
+  SimpleSpanProcessor
```

Comment:
We don't currently run linter on the build, though probably should. But if possible would be good to stick to keeping the trailing comma for all elements of a collection
```suggestion
  SimpleSpanProcessor,
```